src/Common/src/Interop/Windows/advapi32/Interop.GetNumberOfEventLogRecords.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class Advapi32
+    {
+        [DllImport(Libraries.Advapi32, CharSet = CharSet.Unicode, SetLastError = true)]
+        public static extern bool GetNumberOfEventLogRecords(SafeHandle hEventLog, out int NumberOfRecords);
```

Comment:
Use the specific `SafeEventLogReadHandle ` / `SafeEventLogWriteHandle` in all the interop signatures?

Reply:
Ok, thanks. I'm going to fix that. Could you please explain what's going on with Windows x64 Debug Build and UWP CoreCLR x64 Debug Build ?